test/sagemaker_tests/tensorflow/inference/test/integration/local/conftest.py

```diff
@@ -13,7 +13,8 @@
 
 import pytest
 
-FRAMEWORK_LATEST_VERSION = '1.13'
+
+FRAMEWORK_LATEST_VERSION = '2.2'
```

Comment:
If this is being updated in this PR, did it impact previous inference tests?

Reply:
No, it shouldn't have. We are explicitly calling the "--framework_version" command. I can probably remove this and make framework_version a required argument to avoid confusion here